doc/customTags.hbs

```diff
@@ -0,0 +1,3 @@
+{{#if customTags}}{{#each customTags}}{{#unless (isAsync)}}**{{titleCase tag}}**: {{#with (parseType value)~}}
```

Comment:
All symbols, except isAsync, are defined by jsdoc translator and have special meaning. I can add linebreaks and rename isAsync, but that's all.